### PR TITLE
Remove all references to nwalter2

### PR DESCRIPTION
**Pull request by nwalters512**
I noticed that my Illinois email was actually part of the issues UI 😬 I decided to remove all references to my old Illinois NetID while I was cleaning that up.

### CODE REVIEW

File: apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs [hunk 1]
```diff
@@ -83,7 +83,7 @@
                     <td>
                       Shows all issues that were reported by a user ID like <code>USER</code>.
                       <br>
-                      <strong>Example:</strong> <code>user:nwalter2@illinois.edu</code> shows all issues that were reported by <code>nwalter2@illinois.edu</code>.</td>
+                      <strong>Example:</strong> <code>user:student@example.edu</code> shows all issues that were reported by <code>student@example.edu</code>.</td>
```
Comment (mwest1066) [inline apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs:86]: Do we want to be RFC 2606 compliant and use `example.com` instead? (we are doing this in PT, fwiw)

**Review by mwest1066 — APPROVED** [review]

File: apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs (hunk 1 above)
Comment (nwalters512) [reply apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs:86]: Sure! example.edu resolves to a page that looks exactly like example.com, so I didn't even realize it wasn't an "official" example domain.

Comment (nwalters512) [reply apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs]: Done in 9bad16a.

File: docs/saml.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ The full name of the user, e.g. "Joe Smith". This attribute is often named `disp
 
 ### UID
 
-An identifier with an institution-specific suffix, e.g. "jsmith@example.edu". This attribute is often named `eppn`, `eduPersonPrincipalName`, or `urn:oid:1.3.6.1.4.1.5923.1.1.1.6`.
+An identifier with an institution-specific suffix, e.g. "jsmith@example.com". This attribute is often named `eppn`, `eduPersonPrincipalName`, or `urn:oid:1.3.6.1.4.1.5923.1.1.1.6`.
```
Comment (jonatanschroeder) [inline docs/saml.md:15]: It should be fine to use example.edu. https://www.rfc-editor.org/rfc/rfc2606.html

**Pull request merged**